data/upgrades/tech.json

```diff
@@ -147,7 +147,6 @@
         "title": "Deuterium Power Cells",
         "type": "Tech",
         "slots": ["Tech", "Modification"],
-        "charges": { "value": 2, "recovers": 0 },
```

Comment:
The `charges` field was listed twice. This removes just one so it didn't actually remove the charges from the card since the other line is still there (see line 146)